.github/workflows/tests-levels-matrix.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+$testList = shell_exec('php vendor/bin/phpunit --list-tests');
```

Comment:
currently the `--list-tests` feature does not allow filtering by test-groups.

I have opened a feature request about that, so we may can get rid/simplify this script in the future.

https://github.com/sebastianbergmann/phpunit/issues/5702